### ISSUE TITLE
Obey $TELEPRESENCE_ROOT when loading incluster config

### ISSUE THREAD

**Issue by teozkr** [clux/kube-rs#186]
See https://www.telepresence.io/howto/volumes.

This would make it much easier to debug controllers that depend on in-cluster services.

As a workaround, you can use `proot` to fake the mounts:

`proot -b $TELEPRESENCE_ROOT/var/lib/secrets/:/var/lib/secrets cargo run`

**Comment by clux** [comment]
Seems very reasonable. Thank you.

**Comment by teozkr** [comment]
FWIW I'm not sure I think `kube-rs` should have a bunch of special cases for different libraries, but it would be nice to at least have some hook to be able to do so.

**Comment by clux** [comment]
No, for sure. It's hard enough to test special case GCP auth stuff that depend on weird evars as it stands.
